yt_dlp/extractor/facebook.py

```diff
@@ -944,6 +954,10 @@ class FacebookReelIE(InfoExtractor):
             'timestamp': 1637502609,
             'upload_date': '20211121',
             'thumbnail': r're:^https?://.*',
+            'total_comment_count': 21,
+            'unified_reactors_count': 356,
+            'share_count': 81,
```

Comment:
Don't hardcode the values; otherwise the test would fail if someone shares or comments on this video. Just match that they are of type int.